README.md

```diff
@@ -1,5 +1,154 @@
 # Terra Applications
 
-This repo contains applications that can be launched in [Terra]([https://app.terra.bio/).
+This repo contains tooling for launching custom approved apps in [Terra]([https://app.terra.bio/).
 
-Experimental, created as part of DSP Code Jamboree 2020
+**Note this project is under active development, and may not be fully functional yet.**
+
+## Table of Contents
+
+- [Architecture](#architecture)
+- [App Schema](#app-schema)
+- [Supported Apps](#supported-apps)
+- [Launching an App Locally](#launching-an-app-locally)
+- [Launching an App on Terra](#launching-an-app-on-terra)
+- [Helm Chart](#helm-chart)
+
+# Architecture
+
+Apps are defined by a YAML [schema](#app-schema). An "app launcher" (such as Terra) receives a request to launch an app, parses the app YAML file, and invokes the [helm chart](#helm-chart) with the appropriate values to launch the app on a Kubernetes cluster. Once the app is launched, it may be accessed by the end user in a browser via a proxy URL.
```

Comment:
Is that URL specified somewhere in one of the `yaml`s?

Reply:
It's just the regular Leo proxy URL and is specific to whatever's launching the app.. can clarify though.